api/src/messaging/consumers.py

```diff
@@ -132,6 +146,8 @@ async def receive_message(self, text_data_json: dict) -> dict:
         await database_sync_to_async(
             lambda: message_object.tagged_users.add(*tagged_users)
         )()
+        receivers = list(self.room.members.all())
```

Comment:
Wrap in database_sync_to_async. 

Also, you don't need to convert to a list (see comment for create_message_notifications function definition)